src/variant.ts

```diff
@@ -1033,6 +1033,29 @@ export class Togyzkumalak extends Chess {
   }
 }
 
+export class Bestemshe extends Chess {
+  // TODO - move into own class and have variant family
```

Comment:
Actually I think you should be able to start splitting code like described in the comment if you want, now that the build system is there